packages/core/src/lib/oauth/pkce-handler.ts

```diff
@@ -80,7 +80,7 @@ export async function usePKCECodeVerifier(
     codeVerifier: pkce?.value ?? undefined,
     cookie: {
       name: cookies.pkceCodeVerifier.name,
-      value: "",
+      value: pkce?.value ?? undefined,
```

Comment:
This should not be changed, did you run into an issue? if so, please open a separate PR for this

Reply:
Yes, I don't remember the details, but PKCE was not working correctly without it. If I recall correctly, the code was empty in the token exchange, which seemed to be because the cookie was empty. However, I tried just now with the latest, and I did not encounter an issue. I will revert, and open an issue if I come across it again.

Reply:
Oh, I think I encountered the issue again. The pkce challenge was not validated. It seems to be related to checking both "pkce" and "state". I did not encounter the issue when I just checked "pkce". That being said, I don't think this is the most appropriate solution.